laphayen/VisitorQueueSystem
# Create a Spring project for the user queue system

### key updates:

Converted to application.yml:
Migrated configuration from application.properties to application.yml for improved readability and structure.
Set the application server port to 9001 in the new configuration.

Initial Project Setup:
Added base Spring Boot project files and folder structure.
Configured essential dependencies:
- Spring Boot Starter for REST APIs, Thymeleaf templates, and reactive Redis.
- Lombok for cleaner, boilerplate-free code.
- Hibernate Validator for input validation.
- Included Spring Boot DevTools for a faster development workflow.

* * *

### File Changes

Created foundational project files under the src directory.
Added application.yml with server configuration.

* * *

### How to Test

Run ./gradlew bootRun (or gradlew.bat bootRun on Windows).
Verify the application runs on http://localhost:9001.

* * *

This closes #3
